lib/mrb/mrb_ctx.c

```diff
@@ -832,6 +832,28 @@ grn_mrb_ctx_to_exception(mrb_state *mrb)
   return mrb_exc_new_str(mrb, error_class, mrb_str_new_cstr(mrb, message));
 }
 
+mrb_value
```

Comment:
This should not be exported:

```suggestion
static mrb_value
```

Reply:
They should be defined right after `ctx_is_opened()`.